playbooks/roles/filebeat/meta/main.yml

```diff
@@ -0,0 +1 @@
+dependencies: []
```

Comment:
This file was only required for ansible-galaxy, so you can remove it now.

Reply:
:+1: